astropy/cosmology/scalar_inv_efuncs.pyx

```diff
@@ -0,0 +1,225 @@
+""" Cython inverse efuncs for cosmology integrals"""
+#cython boundcheck=False
```

Comment:
If you wanted to go that route, it would probably make more sense to have a single massive neutrino version than a 3-massive neutrino version.  At the moment that's the most common case (the Planck cosmologies, which I assume will become the default some day).  But I felt it was ugly enough having the no gamma, masseless nu, and massive nu cases for each subclass...